rafthttp/transport.go

```diff
@@ -202,11 +202,16 @@ func (t *Transport) Stop() {
 	if tr, ok := t.pipelineRt.(*http.Transport); ok {
 		tr.CloseIdleConnections()
 	}
+	t.peers = nil
+	t.remotes = nil
 }
 
 func (t *Transport) AddRemote(id types.ID, us []string) {
 	t.mu.Lock()
 	defer t.mu.Unlock()
+	if t.remotes == nil {
+		panic("transport stopped")
```

Comment:
OK, I see how that can race if there's no clean shutdown. Will swap panic for return.